splitio/sync/telemetry.py

```diff
@@ -0,0 +1,55 @@
+"""Telemetry Sync Class."""
+
+from splitio.api.telemetry import TelemetryAPI
+from splitio.engine.telemetry import TelemetryStorageConsumer
+
+class TelemetrySynchronizer(object):
```

Comment:
this should be an interface so you can use this interface over the entire factory instead of pointing/using TelemetryAPI or TelemetryStorage, then you will have a TelemetryRedisSynrhonizer and TelemetryIMSynchronizer

Reply:
will do